crates/electrum/src/electrum_ext.rs

```diff
@@ -539,9 +526,8 @@ fn populate_with_spks<K, I: Ord + Clone>(
             }
 
             for tx in spk_history {
-                let tx_entry = update.graph_update.entry(tx.tx_hash).or_default();
```

Comment:
I think you have accidentally removed the ability for us to fetch unconfirmed/mempool transactions.